modules/ROOT/pages/govcloud-data-control.adoc

```diff
@@ -0,0 +1,98 @@
+= GovCloud Data Control
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images/
+
+MuleSoft Government Cloud (GovCloud) organizations can restrict the data entities exposed to their Salesforce Community Cloud ACM instance.
```

Comment:
```suggestion
Your MuleSoft Government Cloud organization can restrict the data entities exposed to its Salesforce Community Cloud API Community Manager instance.
```

Reply:
Never abbr. Anypoint API Community Manager to ACM.
GLOBALLY search for and correct.

Reply:
Fixed.